src/sketchup-stl/exporter.rb

```diff
@@ -261,7 +277,7 @@ def self.do_options
 
         # Row 1 Export Selected
         chk_selection = SKUI::Checkbox.new(
-          'Export selected geometry only.',
+          'Export only current selection',
```

Comment:
I copied the text from the other Pro exporters - I know that doesn't necessarily make it right but it is consistent.